login_server/src/cpp/controller/Group.cpp

```diff
@@ -17,7 +17,7 @@ namespace controller {
 
 	Poco::AutoPtr<Group> Group::create(const std::string& alias, const std::string& name, const std::string& url, const std::string& home, const std::string& description)
 	{
-		auto db = new model::table::Group(alias, name, url, home, description);
+		auto db = new model::table::Group(alias, name, url,"", home, description);
```

Comment:
```suggestion
		auto host = "";
		auto db = new model::table::Group(alias, name, url, host, home, description);
```

Readability over optimization, that's the task of the compiler

Reply:
Please be caution with code suggestions in c++ with auto type. 
In this case I'm not sure if it works like expected because for the compiler it isn't really clear which type he should use. 
"" could be string or char[].
But as this was provisional I have now make it right.